src/constructs/aws/Queue.ts

```diff
@@ -114,21 +115,24 @@ export class Queue extends AwsConstruct {
 
         const maxRetries = configuration.maxRetries ?? 3;
 
+        const baseName = `${this.provider.stackName}-${id}`;
+
         const dlq = new CdkQueue(this, "Dlq", {
-            queueName: `${this.provider.stackName}-${id}-dlq`,
+            queueName: `${baseName}-dlq`,
             // 14 days is the maximum, we want to keep these messages for as long as possible
             retentionPeriod: Duration.days(14),
```

Comment:
We probably need the DLQ to be configured as FIFO as well, right?

Reply:
I wasn't entirely sure what the best practice is there, but thinking about it that does make sense to me. I'm going to update it

Reply:
Ah never mind:

> The dead-letter queue of a FIFO queue must also be a FIFO queue. Similarly, the dead-letter queue of a standard queue must also be a standard queue.